backports/py-rosdistro/force-to-enable-alpine-platform.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/src/rosdistro/distribution_file.py b/src/rosdistro/distribution_file.py
+index c330329..7b35e41 100644
+--- a/src/rosdistro/distribution_file.py
++++ b/src/rosdistro/distribution_file.py
+@@ -74,6 +74,20 @@ class DistributionFile(object):
+                     assert os_code_name not in self.release_platforms[os_name], "Distribution '%s' specifies the os_code_name '%s' multiple times for the os_name '%s'" % (self.name, os_code_name, os_name)
+                     self.release_platforms[os_name].append(os_code_name)
+ 
++        # Tweak for alpine platform
++        self.release_platforms['alpine'] = []
++        if name == 'kinetic':
++            self.release_platforms['alpine'].append('3.7.0')
```

Comment:
add `3.7` for https://github.com/seqsense/aports-ros-experimental/pull/80#pullrequestreview-296606167